pkg/trace/agent/agent_test.go

```diff
@@ -366,82 +366,6 @@ func TestProcess(t *testing.T) {
 	})
 }
 
-func TestFilteredByTags(t *testing.T) {
```

Comment:
Why was this test removed? `filteredByTags` is not removed by this PR.

Reply:
Good catch, added it back.